docs/operating-scylla/procedures/cluster-management/add-node-to-cluster.rst

```diff
@@ -5,49 +5,73 @@ Adding a New Node Into an Existing ScyllaDB Cluster (Out Scale)
 When you add a new node, other nodes in the cluster stream data to the new node. This operation is called bootstrapping and may
 be time-consuming, depending on the data size and network bandwidth. If using a :ref:`multi-availability-zone <faq-best-scenario-node-multi-availability-zone>`, make sure they are balanced.
 
-.. note::
 
-    Adding a new node requires at least a quorum of nodes in a cluster to be available. 
-    If the quorum is lost, it must be restored before a new node is added. 
-    See :doc:`Handling Node Failures </troubleshooting/handling-node-failures>` for details. 
+.. caution::
+
+    The procedure described below applies to clusters where Raft-based topology 
+    is enabled. Raft-based topology is automatically enabled in new clusters.
+
+    If you've upgraded an existing cluster from version 5.4, ensure that you 
+    :doc:`manually enabled Raft-based topology </upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/enable-raft-topology>`.
+    Without Raft-based topology enabled, you must consider the following limitations 
+    while following the procedure:
+
+    * You cannot add new nodes to the cluster if any nodes are down.
```

Comment:
This limitation applies with Raft-based topology too. We haven't gotten rid of it.

You can replace a node if there are some dead nodes but quorum available, but that worked pre-Raft-topology too. (But this is not relevant to this document, which describes regular node bootstrap, not replace; regular bootstrap requires all nodes)